clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestDataRecoveryClient.java

```diff
@@ -52,6 +79,47 @@ private void verifyRecoveryResults(boolean isSuccess) {
     }
   }
 
+  private void estimateRecovery() {
+    EstimateDataRecoveryTimeCommand.Params cmdParams = new EstimateDataRecoveryTimeCommand.Params();
+    planningExecutor = spy(PlanningExecutor.class);
+    controllerClient = mock(ControllerClient.class);
+
+    List<String> mockCmd = new ArrayList<>();
+    mockCmd.add("sh");
+    mockCmd.add("-c");
```

Comment:
As explained, we are not running new command as a child shell process in this case. So, let's remove these.

Reply:
fixed!